pyfr/solvers/base/elements.py

```diff
@@ -176,6 +183,8 @@ def set_backend(self, backend, nscalupts, nonce, linoff):
             self._scal_fpts = salloc('scal_fpts', nfpts)
         if 'scal_qpts' in sbufs:
             self._scal_qpts = salloc('scal_qpts', nqpts)
+        if 'comm_fpts' in sbufs:
```

Comment:
Also, is it not possible to save some storage in the Lobatto case as we never need to allocate vect_fpts?